<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YaChatDlg</class>
 <widget class="QWidget" name="YaChatDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">YaChatDlg, #chatTopFrame {
	background-image: url(:/customwidgets/data/window_background_crop.png);
    background-repeat: repeat-x;
    background-color: #d5e6f3;
    background-attachment: fixed;

    border: 0px solid #99b9d2;
}

.QWidget#separator {
    background-color: #d5e6f3;
}

QFrame#bottomFrame {
    background-color: white;
}

QPushButton, QToolButton {
   color: black;
   border: 0px;
   border-image: url(:/customwidgets/generic/data/button.png) 2px;
}

QPushButton:disabled, QToolButton:disabled {
   color: rgba(0, 0, 0, 128);
}

QPushButton:focus, QToolButton:focus {
   border-image: url(:/customwidgets/generic/data/button_focus.png) 2px;
}

QPushButton:hover, QToolButton:hover {
   border-image: url(:/customwidgets/generic/data/button_hover.png) 2px;
}

QPushButton:hover:focus, QToolButton:hover:focus {
   border-image: url(:/customwidgets/generic/data/button_hover_focus.png) 2px;
}

QPushButton:pressed, QToolButton:pressed {
   border-image: url(:/customwidgets/generic/data/button_pressed.png) 2px;
}
</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="chatTopFrame">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>69</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>69</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>7</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="YaContactAvatarLabel" name="contactUserpic">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="YaContactLabel" name="contactName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Эдмон Дантес</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="YaChatContactStatus" name="contactStatus">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="buttonCall">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Call</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="yaiconsets.qrc">
              <normaloff>:/iconsets/chatwindow/call.png</normaloff>:/iconsets/chatwindow/call.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="contactToolTipArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Info</string>
            </property>
            <property name="icon">
             <iconset resource="yaiconsets.qrc">
              <normaloff>:/iconsets/chatwindow/profile.png</normaloff>:/iconsets/chatwindow/profile.png</iconset>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ChatSplitter" name="chatSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QFrame" name="topFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>255</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
    background-color: white;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="YaChatView" name="chatView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="bottomFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="separator" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>4</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>4</height>
            </size>
           </property>
           <zorder>bottomFrame</zorder>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="LineEdit" name="mle">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="EmoticonsPopupButton" name="buttonEmoticons">
                <property name="minimumSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="YaChatSendButton" name="sendButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>31</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>31</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
        <zorder>separator</zorder>
        <zorder>mle</zorder>
        <zorder>toolButton</zorder>
        <zorder>sendButton</zorder>
        <zorder></zorder>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>chatTopFrame</zorder>
  <zorder></zorder>
  <zorder>chatView</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>YaChatContactStatus</class>
   <extends>QLabel</extends>
   <header>yachatcontactstatus.h</header>
  </customwidget>
  <customwidget>
   <class>YaContactAvatarLabel</class>
   <extends>QFrame</extends>
   <header>yacontactavatarlabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>YaContactLabel</class>
   <extends>QLabel</extends>
   <header>yacontactlabel.h</header>
  </customwidget>
  <customwidget>
   <class>ChatSplitter</class>
   <extends>QSplitter</extends>
   <header>chatsplitter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QTextEdit</extends>
   <header>msgmle.h</header>
  </customwidget>
  <customwidget>
   <class>YaChatView</class>
   <extends>QFrame</extends>
   <header>yachatview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>YaChatSendButton</class>
   <extends>QToolButton</extends>
   <header>yachatsendbutton.h</header>
  </customwidget>
  <customwidget>
   <class>EmoticonsPopupButton</class>
   <extends>QToolButton</extends>
   <header>emoticonspopupbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="yastuff.qrc"/>
  <include location="yaiconsets.qrc"/>
 </resources>
 <connections/>
</ui>
